Sources/LaunchItem.swift

```diff
@@ -44,16 +45,18 @@ public enum LaunchItem: CustomStringConvertible, Equatable {
             return true
         case let (.userActivity(itemLHS), .userActivity(itemRHS)) where itemLHS == itemRHS:
             return true
+        case let (.sourceApplication(itemLHS), .sourceApplication(itemRHS)) where itemLHS == itemRHS:
+            return true
         case (.none, .none):
             return true
-        default:
+        case (.remoteNotification, _), (.localNotification, _), (.openURL, _), (.shortcut, _), (.userActivity, _), (.sourceApplication, _), (.none, _):
             return false
         }
     }
     
     // MARK: Initialization
     
-    init(launchOptions: [AnyHashable : Any]?) {
+    init(launchOptions: [UIApplicationLaunchOptionsKey : Any]?) {
```

Comment:
👏

Reply:
Is this not a problem for someone building against iOS 9?

Reply:
Oh hey... kinda surprised that Xcode isn't warning that we need `#if available` here. Should be easy enough to test how this works. I wonder if `UIApplicationLaunchOptionsKey`'s conformance to `RawRepresentable` is doing some magic here.

Reply:
I guess so?  The sample app runs on iOS 9.3 fine.  Since the earliest Xcode version we support is 8.0, which means a base SDK of iOS 10, it must be using RawRepresentable.